static/js/components/PhotometryPlot.jsx

```diff
@@ -955,14 +957,39 @@ const PhotometryPlot = ({
             legend: {
               orientation: mode === "desktop" ? "v" : "h",
               yanchor: "top",
-              y: mode === "desktop" ? 1 : -0.25,
-              x: mode === "desktop" ? 1.15 : 0,
+              y: mode === "desktop" ? 1 : plotData?.length > 10 ? -0.4 : -0.3, // eslint-disable-line no-nested-ternary
```

Comment:
A comment explaining these decisions would be useful here

Reply:
done!